stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -450,6 +451,8 @@ class FPDF(GraphicsStateMixin):
         new_x: XPos | str = ...,
         new_y: YPos | str = ...,
         wrapmode: WrapMode = ...,
+        dry_run: bool = False,
+        output: MethodReturnValue = ...,
```

Comment:
It's passed to `.coerce()` here before being used for anything, so I think we can be slightly more permissive: https://github.com/PyFPDF/fpdf2/blob/37fc30f14481a56ab362dbca3dd6e4c45ab8f043/fpdf/fpdf.py#L3628

```suggestion
        output: MethodReturnValue | str | int = ...,
```